src/ui.text-alert/index.stories.js

```diff
@@ -12,36 +13,46 @@ export default {
   id: "4030",
   title: "Text & Content / Alert",
   component: UAlert,
-  args: {
-    text: "UHint",
-    slotDefaultTemplate: `
-      <template #default>
-        <p>
-          <b>Lorem ipsum dolor sit amet,</b>
-          <u>consectetur adipiscing elit,</u>
-          <em>sed do eiusmod tempor incididunt
-          ut labore et dolore magna aliqua.</em>
-          <a href="https://uk.wikipedia.org/wiki/Lorem_ipsum" target="_blank">Wikipedia</a>
-        </p>
-      </template>
-    `,
-  },
+  args: {},
   argTypes: {
     ...getArgTypes(UAlert.name),
   },
 };
 
+const defaultTemplate = `
+  <UText size="size">
```

Comment:
No, **UText** need to be implemented inside of **UAlert**, not in the story. 🤦‍♂️